src/ui/utils/environment.ts

```diff
@@ -141,5 +141,7 @@ export function getDisplayedUrl(url: string) {
 }
 
 export function getSystemColorSchemePreference() {
-  return window.matchMedia("(prefers-color-scheme:dark)") ? "dark" : "light";
+  const prefersDarkTheme = window.matchMedia("(prefers-color-scheme:dark)").matches
```

Comment:
Small lint fix — this needs a `;` at the end. Running `trunk check` should also take care of it!

Reply:
fixed it @jaril . Also the above mentioned issue by @john-msftech seems like a different issue because as of now, whenever theme is set to system it gets changed dark or light based on system default theme. If we want to listen to system theme change then we need to keep theme set to system only and not change it to dark/light 🤔